src/libraries/Common/src/Interop/Windows/Advapi32/Interop.GetTokenInformation.cs

```diff
@@ -9,39 +9,24 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-#if DLLIMPORTGENERATOR_ENABLED
         [GeneratedDllImport(Interop.Libraries.Advapi32, SetLastError = true)]
         internal static partial bool GetTokenInformation(
-#else
-        [DllImport(Interop.Libraries.Advapi32, SetLastError = true)]
-        internal static extern bool GetTokenInformation(
-#endif
             SafeAccessTokenHandle TokenHandle,
             uint TokenInformationClass,
             SafeLocalAllocHandle TokenInformation,
             uint TokenInformationLength,
             out uint ReturnLength);
 
-#if DLLIMPORTGENERATOR_ENABLED
         [GeneratedDllImport(Interop.Libraries.Advapi32, SetLastError = true)]
         internal static partial bool GetTokenInformation(
-#else
-        [DllImport(Interop.Libraries.Advapi32, SetLastError = true)]
-        internal static extern bool GetTokenInformation(
-#endif
             IntPtr TokenHandle,
             uint TokenInformationClass,
             SafeLocalAllocHandle TokenInformation,
             uint TokenInformationLength,
             out uint ReturnLength);
 
-#if DLLIMPORTGENERATOR_ENABLED
-        [GeneratedDllImport(Interop.Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        [GeneratedDllImport(Interop.Libraries.Advapi32, SetLastError = true)]
```

Comment:
why remove CharSet = CharSet.Unicode ? is it the default now?

Reply:
It is not the default. I removed it because it is not necessary for this p/invoke - no string data being marshalled, no A/W suffix variants - and having it set to Unicode in this case makes it so that we do an extra probe for an entry point named `GetTokenInformationW`, since when ExactSpelling is false, we check for the W suffix first for Unicode (but no suffix first for Ansi).

Reply:
this seems like a great thing for the generator to warn on (I say the generator since it's new so it wouldn't be a breaking change, and folks are touching their sources anyway)

Reply:
We're currently discussing how we can make the whole character set situation better with the generator approach: https://github.com/dotnet/runtime/issues/61326.

One thing we are considering as part of that is removing ExactSpelling (so weird/negative implications of char set like this wouldn't be a thing) and effectively requiring it to always be true.